webui/react/src/styles/icons.scss

```diff
@@ -4,8 +4,8 @@
   font-style: normal;
   font-weight: normal;
   src:
-    url('fonts/determined-ai.woff2?wo5opj') format('woff2'),
-    url('fonts/determined-ai.woff?wo5opj') format('woff');
+    url('fonts/determined-ai.woff2?wo6opj') format('woff2'),
```

Comment:
@hkang1 not sure if there are rules to change this hash?

Reply:
we change the icon font so infrequently I just use the icomoon generated hash to replace it directly